README.rst

```diff
@@ -33,6 +33,7 @@ Apart from the problem, options can be passed to ``benchopt run``, to restrict t
 .. code-block::
 
 	$ benchopt run benchmark_bci -s solver1 -d dataset2 --max-runs 10 --n-repetitions 10
+    $ BENCHOPT_DEBUG=1 benchopt run . -s MDM -d BNCI -r 1 -n 1
```

Comment:
Not sure we want to recommend `DEBUG` mode? And usually, we consider that the user just cloned the benchmark so they are in the parent directory.
```suggestion
    $ benchopt run benchmark_bci -s MDM -d BNCI -r 1 -n 1
```